### DISCUSSION TITLE
Column type upload not showing the correct url

### DISCUSSION THREAD

**Discussion by christmex** [Laravel-Backpack/community-forum#551] [Q&A (Help)] [answered]
Hi guys so I'm performing file upload in my backpack v6 but I got something wrong the URL in the table is not correct, here are my backpack version details
![image](https://github.com/Laravel-Backpack/community-forum/assets/34405837/d8ddae5d-8f08-445a-9e6c-10a5726f965e)

I'm using this in my **setupUpdateOperation**
```php
//inside my setupUpdateOperation
CRUD::field('file_scan')
    ->type('upload')
    ->withFiles([
        'disk' => 'public', // the disk where file will be stored
        'path' => 'file-scan', // the path inside the disk where file will be stored
]);
```

then in my **setupListOperation**
```php
//setupListOperation
CRUD::addColumn([
    'name'   => 'file_scan',
    'type'   => 'upload',
    'label'  => 'File Scan',// the disk where file will be stored
    'disk' => 'public',
]);
```

it does not add the storage prefix, so it's not the correct URL for accessing my file
![image](https://github.com/Laravel-Backpack/community-forum/assets/34405837/9425944c-a4dc-419b-98f3-1d9e85860e80)

but when I go to the edit page, the URL is valid
![image](https://github.com/Laravel-Backpack/community-forum/assets/34405837/9c74d03f-1d19-4537-85a4-7f39622a6431)

Let me know if you need another detail, sorry for my English, how to fix this problem?

**Comment by pxpm** [comment, marked as answer]
Hey @christmex I could reproduce it. Thanks for the report 🙏 

I've submitted https://github.com/Laravel-Backpack/CRUD/pull/5179 to address this! 

If you happen to give it a test let me know if it worked as expected for you. 

Cheers

**Reply by christmex** [reply]
Sure, thanks @pxpm  🙏🙌, but the PR hasn't merged yet, I'll be back after the update is available

**Reply by christmex** [reply]
Hi @pxpm I just giving a shot at the latest version of backpack, and it works, thank you so much 🙌🙏
